components/alert/AlertMap.vue

```diff
@@ -711,250 +867,181 @@ export default Vue.extend({
     },
     async loadRegions(): Promise<void> {
       if (this.mapRegions.size !== 0) {
-        return;
+        return
       }
 
-      const regions = await new MapRegionDataRequest()
-        .pull(this.alert.zone ? this.alert.zone : Zone.INDAR);
-      
-      this.$L.Icon.Default.prototype.options.iconAnchor = [0, 0];
-      this.$L.Icon.Default.prototype.options.tooltipAnchor = [0, 0];
+      const regions = await new MapRegionDataRequest().pull(
+        this.alert.zone ? this.alert.zone : Zone.INDAR
```

Comment:
If we don't have the alert zone here we have bigger problems?

Reply:
I think this was a concession to the TS linter yelling at me about "oh this could technically be undefined!"

Reply:
Hmm I tend to do a guard around stuff like that, like this:

```
if (!this.alert.zone) {
  console.error('No zone!');
  return
}
```
Tends to keep TS happy, plus it stops wonkyness further down the line happening, as you're then dealing with unknown data being set by wonky circumstances, I prefer to die early and abort execution rather than supplanting / getting around TS with weirdness.

Reply:
Done 👍